src/charm.py

```diff
@@ -135,8 +137,9 @@ def _labels(self):
                 invalid_labels.append(f"{key}:{value}")
 
         if invalid_labels:
-            self.unit.status = BlockedStatus(f"Invalid labels : {invalid_labels}")
-
+            logger.error("Invalid labels : %s", invalid_labels)
+            logger.error("Valid labels must be key:value pairs")
```

Comment:
Labels must be specified in key:value format

Reply:
done